common/common.cc

```diff
@@ -279,22 +282,30 @@ void appendFilesInDir(string_view basePath, const string &path, const sorbet::Un
             if (dotLocation != string::npos) {
                 string_view ext(fullPath.c_str() + dotLocation, fullPath.size() - dotLocation);
                 if (extensions.contains(ext)) {
-                    result.push_back(move(fullPath));
+                    // Remove the ./ prefix in the final path (but it's convenient to have it
+                    // above for the sake of matching ignore patterns). Also don't allocate unless we have to.
+                    if (basePath == ".") {
+                        result.push_back(fullPath.substr(2));
```

Comment:
What about using `fullPath.erase(0, 2)` instead of copying to make the substring?